### PR TITLE
Updated Quiz and QuizApplication models

### PR DESCRIPTION
**Pull request by NematovaH**

### CODE REVIEW

File: src/Arcana.Domain/Entities/Quizzes/Quiz.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace Arcana.Domain.Entities.Quizzes;
+
+public class Quiz : Auditable
+{
+    public string Name { get; set; }
+    public int QuestionCount { get; set; }
+    public DateTime StartTime { get; set; }
+    public DateTime EndTime { get; set; }
+    public long ModuleId { get; set; }
```
Comment (makhammadsoliyev) [inline src/Arcana.Domain/Entities/Quizzes/Quiz.cs:11]: Add module entity under ModuleId

File: src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using Arcana.Domain.Commons;
+
+namespace Arcana.Domain.Entities.QuizApplications;
+
+public class QuizApplication : Auditable
+{
+    public long StudentId { get; set; }
```
Comment (makhammadsoliyev) [inline src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs:9]: Add Studdent Entity under studentId

File: src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    public long QuizId { get; set; }
```
Comment (makhammadsoliyev) [inline src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs:11]: Add quiz entity under quizid

**Review by makhammadsoliyev — CHANGES_REQUESTED** [review]
Follow feedbacks

File: src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs (hunk 2 above)
Comment (NematovaH) [reply src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs:9]: ok

File: src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs (hunk 3 above)
Comment (NematovaH) [reply src/Arcana.Domain/Entities/QuizApplications/QuizApplication.cs:11]: ok

**Pull request closed without merging**